foxtrot-common/src/main/java/com/flipkart/foxtrot/common/query/QueryResponse.java

```diff
@@ -28,12 +28,18 @@
  */
 public class QueryResponse implements ActionResponse {
     private List<Document> documents;
+    private long totalHits;
 
     public QueryResponse() {
     }
 
     public QueryResponse(List<Document> documents) {
```

Comment:
@santanusinha I am not asking for moving it to action response. I am asking to completely remove this method